base/inc/fields/js/date-range-field.js

```diff
@@ -1,23 +1,40 @@
-/* global jQuery, soWidgets */
+/* global jQuery, pikaday */
 
 (function( $ ) {
 	$( document ).on( 'sowsetupformfield', '.siteorigin-widget-field-type-date-range', function ( e ) {
+		var $dateRangeField = $( this );
+		var valField = $dateRangeField.find( 'input[type="hidden"][class="siteorigin-widget-input"]' );
+		
+		if ( $dateRangeField.data( 'initialized' ) ) {
+			return;
+		}
 
-		var valField = $( this ).find( 'input[type="hidden"][class="siteorigin-widget-input"]' );
-
-		if ( $( this ).find( '[class*="sowb-specific-date"]' ).length > 0 ) {
+		if ( $dateRangeField.find( '[class*="sowb-specific-date"]' ).length > 0 ) {
 			var createPikadayInput = function ( inputName, initVal ) {
-				var $field = $( this ).find( '.' + inputName + '-picker' );
+				var $field = $dateRangeField.find( '.' + inputName + '-picker' );
 				var picker = new Pikaday( {
 					field: $field[ 0 ],
 					blurFieldOnSelect: false,
+					toString( date, format ) {
```

Comment:
For consistency, we could stick to `toString: function(...) {` here

Reply:
👍